### PR TITLE
micro: add Russian translation

### PR DESCRIPTION
**Pull request by NikonP**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @NikonP, I left you a comment.

File: pages/linux/aplay.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# aplay
```
Comment (navarroaxel) [inline pages/linux/aplay.md:1]: Please @NikonP remove this page from this PR.

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @NikonP

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good to me.

**Comment by sbrl** [comment]
Feel free to open 1 big PR if you're translating lots of pages, with 1 commit per page  :-)

**Pull request merged**